act-framework.bs

```diff
@@ -35,7 +35,13 @@ ACT Rule Structure {#structure}
 Rule Outline {#structure-outline}
 -----------------------
 
-Editor note: This sections gives a broad outline of parts make up an ACT Rule. We'll go into further detail in sections below. At this point it is important to show the reader the big picture
+A rule MUST provide the following items written in plain language:
+* A descriptive title 
```

Comment:
Should we add, pass / fail criteria to the rule outline or do we expect that to be included in the test procedure.